firefox-ios/Shared/Supporting Files/ia.lproj/EnhancedTrackingProtection.strings

```diff
@@ -41,10 +41,10 @@
 "Menu.EnhancedTrackingProtection.ClearData.AlertText.v128" = "Remover cookies e datos de sito pro %@ pote disconnecter te de sitos web e vacuar carrettos de compras.";
 
 /* The title for the clear cookies and site data alert inside the enhanced tracking protection screen. */
-"Menu.EnhancedTrackingProtection.ClearData.AlertTitle.v128" = "Clarar le cookies e le datos de sito";
+"Menu.EnhancedTrackingProtection.ClearData.AlertTitle.v128" = "Rader le cookies e le datos de sito";
 
 /* The title for the clear cookies and site data button inside the enhanced tracking protection screen. */
-"Menu.EnhancedTrackingProtection.ClearData.ButtonTitle.v128" = "Clarar le cookies e le datos de sito";
+"Menu.EnhancedTrackingProtection.ClearData.ButtonTitle.v128" = "Rader le cookies e le datos de sito";
```

Comment:
@Delphine Just wanted to flag this, I've noticed an existing string seems to have changed. Is this ok?

Reply:
Thanks for asking. Yes, it's expected that some strings get updated by localizers throughout time, to adapt and improve their translations (for example)